cosi/cosi_new.go

```diff
@@ -0,0 +1,130 @@
+package cosi
+
+import (
+	"crypto/cipher"
+
+	"github.com/dedis/kyber/abstract"
+)
+
+func Commit(suite abstract.Suite, s cipher.Stream) (abstract.Scalar, abstract.Point) {
+	// compute random scalar
+	// compute commitment
+	// return both
+}
+
+func AggregateCommitments(suite abstract.Suite, commitments []abstract.Point, mask *Mask) (abstract.Point, *Mask) {
```

Comment:
After discussion with @nikkolasg we came to the conclusion that we need to provide a slice of `*Mask` as input instead of just a single one as each commitment comes with its own mask.